src/applications/claims-status/components/TravelClaimsSection.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import recordEvent from '@department-of-veterans-affairs/platform-monitoring/record-event';
+
+export default function TravelClaimsSection() {
+  const recordLinkClick = () =>
+    recordEvent({
+      event: 'nav-link-click',
+      'link-label': 'Review and file travel claims',
+      'link-destination': '/my-health/travel-pay/claims',
+    });
+
+  return (
+    <div className="vads-u-margin-y--4">
+      <h2 id="your-travel-claims">Your travel claims</h2>
```

Comment:
When I run this branch locally I do see the travel claims section, but not the link in the "On this page" component.  Is there anything I need to do to make it show?

<img width="1119" alt="Screenshot 2025-05-06 at 10 53 22 PM" src="https://github.com/user-attachments/assets/cd680843-0d97-4469-9222-d68e03ea8e8a" />

Reply:
🤦‍♀️  Past my bedtime. Thanks for humoring me!